resources/views/admin.blade.php

```diff
@@ -177,7 +231,24 @@ function deleteEmployee(id){
                     });
                 });
 
+                // Clear Forms on Submit
+                $('.btn-close').click(function (e) {
+                    $('.form-modal').trigger("reset");
+
+                });
+
+                // Set Branch on Selct
+                $('#select-create').on('click' , function (e) {
```

Comment:
Please explain this code blocks @axnish27

Reply:
This code is for to fill the company branch when the company is selected....I have attached the behavior as clip, please find the attachment



https://github.com/user-attachments/assets/ac314cec-ff74-4f6b-98d4-31fddee733e1